frontend/src/App.js

```diff
@@ -45,4 +57,13 @@ class App extends React.Component<InternalProps> {
     }
 }
 
-export default connect()(App);
+const mapStateToProps = (state: Object): Props => {
+    const { l10n, locales } = state;
+
+    return {
+        l10n,
+        locales,
+    };
+};
```

Comment:
I think it's better if we don't hard-code the module names, and instead use their declared `NAME` constant. That will allow us to refactor easier in the future. Take a look at the way I did it in other files, for example here: https://github.com/mozilla/pontoon/blob/master/frontend/src/modules/search/components/SearchBox.js#L104

Note that `router` is an exception because it's an external library and I don't expect it to ever change.